src/escapei.rs

```diff
@@ -112,7 +112,14 @@ fn _escape<F: Fn(u8) -> bool>(raw: &str, escape_chars: F) -> Cow<str> {
             b'\'' => escaped.extend_from_slice(b"&apos;"),
             b'&' => escaped.extend_from_slice(b"&amp;"),
             b'"' => escaped.extend_from_slice(b"&quot;"),
-            _ => unreachable!("Only '<', '>','\', '&' and '\"' are escaped"),
+
+            b'\t' => escaped.extend_from_slice(b"&#9;"),
+            b'\r' => escaped.extend_from_slice(b"&#10;"),
+            b'\n' => escaped.extend_from_slice(b"&#13;"),
+            b' ' => escaped.extend_from_slice(b"&#32;"),
```

Comment:
Why escape these characters?

x09, x0D, x0A are all explicitly allowed as per https://en.wikipedia.org/wiki/Valid_characters_in_XML#Non-restricted_characters

And space is obviously OK